openvoxserver/docker-entrypoint.d/20-use-templates-initially.sh

```diff
@@ -10,7 +10,7 @@ TEMPLATES=/var/tmp/puppet
 cd /etc/puppetlabs/puppet
 for f in auth.conf hiera.yaml puppet.conf puppetdb.conf
 do
-    test -f "$TEMPLATES/$f" && cp -np "$TEMPLATES/$f" .
+    test -f "$TEMPLATES/$f" && cp -n --update=none "$TEMPLATES/$f" .
```

Comment:
You should keep `-p`
```suggestion
    test -f "$TEMPLATES/$f" && cp -p --update=none "$TEMPLATES/$f" .
```

Reply:
Autsch, you're right of course.